api/app/signals/apps/signals/managers.py

```diff
@@ -17,6 +17,7 @@
 update_reporter = DjangoSignal(providing_args=['signal_obj', 'reporter', 'prev_reporter'])
 update_priority = DjangoSignal(providing_args=['signal_obj', 'priority', 'prev_priority'])
 create_note = DjangoSignal(providing_args=['signal_obj', 'note'])
+update_type = DjangoSignal(providing_args=['signal_obj', 'previous_type', 'type'])
```

Comment:
nitpick

These are just named arguments, but do you have a reason to reverse the order of the argument names (prev, current versus current, prev)?

Reply:
Changed order and naming to prev_type